ci-operator/infra/openshift/release-controller/repos/ocp-4.2-openstack.repo

```diff
@@ -0,0 +1,10 @@
+[rhel-openstack]
+name = rhel-openstack
+baseurl = https://use-mirror1.ops.rhcloud.com/enterprise/reposync/4.1/rhel-openstack-rpms/
+          https://use-mirror2.ops.rhcloud.com/enterprise/reposync/4.1/rhel-openstack-rpms/
```

Comment:
this should be `4.2` - if you aren't mirroring to 4.2, we need to get that fixed

Reply:
Here's the reason: https://github.com/openshift/release/blob/d6ea197bb29b214f3ba5b6eca8826db378340a81/ci-operator/infra/openshift/release-controller/repos/ocp-4.2-default.repo#L3-L4. I assumed ocp-4.2.default.repo got it figured out.

Reply:
Should I still change it?

Reply:
Can you open a bug to the ART team that they need to sync 4.2 repos to mirror.openshift.com?